gitlab/payload.go

```diff
@@ -139,7 +139,7 @@ type BuildEventPayload struct {
 	BuildStatus       string      `json:"build_status"`
 	BuildStartedAt    customTime  `json:"build_started_at"`
 	BuildFinishedAt   customTime  `json:"build_finished_at"`
-	BuildDuration     int64       `json:"build_duration"`
+	BuildDuration     float64     `json:"build_duration"`
```

Comment:
I have some doubts here @zrochler in terms of backward compatibility... Due to documentation you should be right, but is it possible, that smbd already used this payload in old way...

Reply:
@Toshik1978 you're probably right. I don't remember what specifically led me to that change and whether it was because I saw it misbehaving, but I undid that change to avoid breaking anyone.

Reply:
@zrochler Zach, but it's a good question actually. Do we have float or int in real payload?
I'm preparing v6 branch at this moment. So we can do backward incompatible changes during 6.0.0 release.